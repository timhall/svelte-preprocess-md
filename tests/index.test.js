const { preprocess } = require('svelte/compiler');
const md = require('../');

it('should preprocess markdown', async () => {
  const output = await preprocess(
    `
# Hello {name}!

<script>
  let name = 'World';
</script>
    `,
    Object.assign(md(), { filename: 'test.md' })
  );

  expect(output.toString()).toMatchSnapshot();
});
