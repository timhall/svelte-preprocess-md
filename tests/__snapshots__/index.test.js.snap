// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should preprocess markdown 1`] = `
"<h1>Hello {name}!</h1>
<script>
  let name = 'World';
</script>
"
`;
